ui/lib/core/addon/components/replication-header.js

```diff
@@ -3,7 +3,9 @@ import layout from '../templates/components/replication-header';
 
 export default Component.extend({
```

Comment:
would you be willing to add JSDoc comments for this component? there are a lot of very similar properties in here and i think it'll help us later to remember what each thing is.